### PR TITLE
🌌 Withdrawal with portfolio shares

### PR DESCRIPTION
**Pull request by skalermo**

### CODE REVIEW

File: test/ragnarok/ManagedPortfolio.test.ts [hunk 1]
```diff
@@ -135,12 +135,13 @@ describe('ManagedPortfolio', () => {
       await depositIntoPortfolio(100)
 
       await expect(portfolioAsLender.withdraw(parseShares(50)))
-        .to.be.revertedWith('ManagedPortfolio: Cannot withdraw when Portfolio is not closed.')
+        .to.be.revertedWith("ManagedPortfolio: Cannot withdraw when Portfolio is not closed")
```
Comment (krzysztof-jelski) [inline test/ragnarok/ManagedPortfolio.test.ts:138]: Isn't the linter rule to have single quotes?

Comment (vanruch) [inline test/ragnarok/ManagedPortfolio.test.ts:138]: lint will fail

Comment (vanruch) [reply test/ragnarok/ManagedPortfolio.test.ts:138]: and same later

**Review by krzysztof-jelski — APPROVED** [review]

File: test/ragnarok/ManagedPortfolio.test.ts (hunk 1 above)
Comment (skalermo) [reply test/ragnarok/ManagedPortfolio.test.ts:138]: I'm on it

**Pull request merged**